src/spec.cr

```diff
@@ -94,6 +95,21 @@ module Shards
             read_mapping(pull) { dependency[pull.read_scalar] = pull.read_scalar }
             development_dependencies << dependency
           end
+        when "targets"
+          read_mapping(pull) do
+            target = Target.new(pull.read_scalar)
+            read_mapping(pull) do
+              case pull.read_scalar
+              when "main"
+                target.main = pull.read_scalar
+              when "options"
```

Comment:
Let's avoid `options` for now; they could be passed as options to the command for example:

```
$ shards build --release
```